modules/userId/eids.js

```diff
@@ -110,6 +110,10 @@ const USER_IDS_CONFIG = {
         third: data.third
       } : undefined;
     }
+  },
+  'haloId': {
+    source: 'audigent.com',
```

Comment:
if you guys look at the liveintent object, segments are coming on the eid. does it make sense for yours to mirror this structure?

Reply:
we intend to add segment functionality to our id endpoint with a further update.. this will allow for better integration with our rtd module and allow for fewer requests